src/installation/live-images/prep.md

```diff
@@ -30,16 +46,25 @@ Write the live image to the device:
 # dd bs=4M if=/path/to/void-live-ARCH-DATE-VARIANT.iso of=/dev/sdX
 ```
 
-Ensure all data is flushed before disconnecting the device:
+dd won't print anything until it's completed (or if it failed), so depending on
+the device, this can take a few minutes or longer.
+
+Finally, ensure all data is flushed before disconnecting the device:
 
 ```
 $ sync
+90+0 records in
+90+0 records out
+377487360 bytes (377 MB, 360 MiB) copied, 0.461442 s, 818 MB/s
 ```
 
-## Create a bootable USB drive or SD card on Windows
+The number of records, amount copied, and rates will all vary depending on the
+device and the live image you chose.
+
+## Create a Bootable USB Drive or SD Card on Windows
 
 The [USBWriter](https://sourceforge.net/projects/usbwriter/) program is the only
```

Comment:
I'm really not sure where this recommendation came from, I certainly don't recommend it and I maintain the images.  Lets either drop it entirely or come up with an actually recommended tool.

Reply:
This seems fine to me. I can't really make any recommendations for Windows, so I'm not sure where it came from. Given that there's the Windows Subsystem for Linux now, it may be feasible to just use dd on Windows 10 and later.